app/brave_generated_resources.grd

```diff
@@ -898,6 +898,10 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_IPFS_ALWAYS_START_INFOBAR_NO" desc="Ignore proposition">
         Cancel
       </message>
+      <!-- Brave Wallet -->
+      <message name="IDS_UTILITY_PROCESS_WALLET_UTILS_NAME" desc="The utility process running wallet utils">
+        Brave Wallet Utils Service
```

Comment:
Personally, I'd rather spell it out if the length is not an issue: `Brave Wallet Utility Service`